synapse/models/geopol.py

```diff
@@ -43,29 +51,63 @@ def getModelDefs(self):
                     ('pol:country', {}, (
                         ('flag', ('file:bytes', {}), {
                             'doc': 'A thumbnail image of the flag of the country.'}),
+
                         ('iso2', ('pol:iso2', {}), {}),
+
                         ('iso3', ('pol:iso3', {}), {}),
+
                         ('isonum', ('pol:isonum', {}), {}),
+
                         ('pop', ('int', {}), {
                             'deprecated': True,
                             'doc': 'Deprecated. Please use :vitals::population.'}),
+
                         ('tld', ('inet:fqdn', {}), {}),
 
                         ('name', ('geo:name', {}), {
                             'doc': 'The name of the country.'}),
+
                         ('names', ('array', {'type': 'geo:name', 'uniq': True, 'sorted': True}), {
                             'doc': 'An array of alternate or localized names for the country.'}),
+
                         ('government', ('ou:org', {}), {
                             'doc': 'The ou:org node which represents the government of the country.'}),
+
                         ('place', ('geo:place', {}), {
                             'doc': 'A geo:place node representing the geospatial properties of the country.'}),
+
                         ('founded', ('time', {}), {
                             'doc': 'The date that the country was founded.'}),
+
                         ('dissolved', ('time', {}), {
                             'doc': 'The date that the country was dissolved.'}),
+
                         ('vitals', ('pol:vitals', {}), {
                             'doc': 'The most recent known vitals for the country.'}),
                     )),
+                    ('pol:immegration', {}, (
+
+                        ('contact', ('ps:contact', {}), {
+                            'doc': 'The contact information for the immigration status record.'}),
+
+                        ('country', ('pol:country', {}), {
```

Comment:
Do we care about the country the individual emigrated **from** ? E.g., :country:from / :country:to ?

Reply:
Any "giving up of status" should be represented by another node. I don't think a "from" country as a singular property is a thing. :+1: